<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>823</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>28</width>
    <height>28</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Box2DWidget" name="gameWindow" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>721</height>
     </rect>
    </property>
    <widget class="QPushButton" name="tortillasButton">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>300</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="chipsButton">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>290</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ramenButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>230</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="noodlesButton">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>150</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="sauceButton">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>220</y>
       <width>21</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="potatoButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="chickenButton">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>500</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="lettuceButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>650</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="mayoButton">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>490</y>
       <width>41</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="sourCreamButton">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>570</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cheeseButton">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>640</y>
       <width>41</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="salsaButton">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>560</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="oilButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>90</y>
       <width>31</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="waterButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>420</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="platesButton">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>370</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bowlsButton">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>360</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="phoneButton">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>160</y>
      <width>231</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">&quot;QPushButton { background-color: transparent }&quot;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="phoneRecipe">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>230</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Box2DWidget</class>
   <extends>QWidget</extends>
   <header location="global">box2dwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
